LayoutTests/imported/w3c/web-platform-tests/webcodecs/audio-decoder.https.any.js

```diff
@@ -2,25 +2,50 @@
 // META: script=/webcodecs/utils.js
 
 const invalidConfigs = [
+  {
+    comment: 'Missing codec',
+    config: {
+      sampleRate: 48000,
+      numberOfChannels: 2,
+    },
+  },
   {
     comment: 'Empty codec',
-    config: {codec: ''},
+    config: {
+      codec: '',
+      sampleRate: 48000,
+      numberOfChannels: 2,
+    },
   },
   {
-    comment: 'Unrecognized codec',
-    config: {codec: 'bogus'},
+    comment: 'Missing sampleRate',
+    config: {
+      codec: 'opus',
+      sampleRate: 48000,
+    },
```

Comment:
This makes no sense. `Missing sampleRate` with a config that has a `sampleRate` value... ?